CHANGELOG.md

```diff
@@ -18,6 +18,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Security
 - No security issues fixed!
 
+## [2.0.0] - 2021-05-01
+### Changed
+- Changed repo ownership to [hyperdevs-team](https://github.com/hyperdevs-team). Thanks [bq](https://github.com/bq) for all the work!
+- Changed package names from `com.bq.*` to `com.hyperdevs.*`
+
+## [1.5.0] - 2021-03-15
+### Added
+- Upgrade Kotlin to 1.4.31 and Kodein to 7.4.0, apart from other Android dependencies.
+
```

Comment:
What do you think is the best option here? To keep the versions released in the más movil repo or to include only a new 2.0.0 version?